app/helpers/application_helper/button/transform_vm_button.rb

```diff
@@ -0,0 +1,12 @@
+class ApplicationHelper::Button::TransformVmButton < ApplicationHelper::Button::Basic
+  needs :@record
+
+  def visible?
+    @record.vendor == "vmware"
+  end
+
+  def disabled?
+    # Is there a provider that supports import?
+    EmsInfra.all.select(&:validate_import_vm).empty?
+  end
```

Comment:
how expensive is this operation? I see it's going to be run each time this button is to be displayed...

Also I wonder if this could make use of the `supports?` feature that we already have on the backend.